### DISCUSSION TITLE
Peak sitemap on Statamic

### DISCUSSION THREAD

**Discussion by opequin** [studio1902/statamic-peak#399] [Q&A]
Hello

I have a website under Statamic.
On my dashboard in Global / SEO, I have chosen :  "Enable the Peak Sitemap".
But I don't find the url of this peak sitemap. 
Do you know where it is (which url ?) ?

There is nothing in mywebsite.com/sitemap.xml

Any idea what the right url is for the Peak Sitemap ?

**Comment by robdekort** [comment]
Hey there, yeah that should be the correct URL. It shows a sitemap here. What do you see? Have you configured the proper collections that need to go into your sitemap? For example:

<img width="441" alt="Screenshot 2024-09-06 at 21 12 48" src="https://github.com/user-attachments/assets/20ef561f-f4a8-4a06-ada3-23a3f5b876ad">

**Comment by NielsCS2025** [comment]
Hello
when I go to the mywebsite.com/sitemap.xml I get server error 500. 
When I go to mywebsite.com/sitemaps.xml i see this:
`<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
<sitemap>
<loc>https://mysite.com/sitemap.xml</loc>
</sitemap>
</sitemapindex>`


I have colletions enable to show in the sitemap. What am I doing wrong?

**Reply by robdekort** [reply]
What's the error in your laravel.log?

**Reply by NielsCS2025** [reply]
I asked my devs and they don't see any that directly has to do with this. 
This is the only one: 

> SQLSTATE[HY001]: Memory allocation error: 1038 Out of sort memory, consider increasing server sort buffer size (Connection: mysql, SQL: select * from `entries` where `collection` in (pages) and `published` = 1 and (json_extract(`data`, '$."redirect"') is null OR json_type(json_extract(`data`, '$."redirect"')) = 'NULL') and json_extract(`data`, '$."seo_noindex"') != true order by `order` asc)
Illuminate\\Database\\QueryException(code: HY001: SQLSTATE[HY001]: Memory allocation error: 1038 Out of sort memory, consider increasing server sort buffer size (Connection: mysql, SQL: select * from `entries` where `collection` in (pages) and `published` = 1 and (json_extract(`data`, '$.\"redirect\"') is null OR json_type(json_extract(`data`, '$.\"redirect\"')) = 'NULL') and json_extract(`data`, '$.\"seo_noindex\"') != true order by `order` asc) at /home/ploi/marketing.caresquare.com/vendor/laravel/framework/src/Illuminate/Database/Connection.php:823)

**Reply by robdekort** [reply]
Apparently your server runs out of memory generating the sitemap. Big site?

**Reply by NielsCS2025** [reply]
Not really. max 50pages.

**Reply by robdekort** [reply]
Well, it's a memory issue as the error suggests. I know nothing about your server, but upping it is how you can resolve this.

**Reply by NielsCS2025** [reply]
will this fix the sitemap issue?
